hnn_core/network.py

```diff
@@ -161,13 +165,13 @@ def __init__(self, params, add_drives_from_params=False,
         # Create CellResponse object, initialised with simulation time points
         self.cell_response = CellResponse(times=times)
 
-        # Source list of names, first real ones only!
-        self.cellname_list = [
-            'L2_basket',
-            'L2_pyramidal',
-            'L5_basket',
-            'L5_pyramidal',
-        ]
+        # Source dict of names, first real ones only!
+        self.cell_types = {
+            'L2_basket': basket((0, 0, 0), cell_name=_short_name('L2_basket')),
```

Comment:
```suggestion
            'L2_basket': basket((0, 0, 0), _short_name('L2_basket')),
```
My comments above were related to making sure this won't break anything